tools/packaging/scripts/configure-hypervisor.sh

```diff
@@ -445,7 +454,9 @@ generate_qemu_options() {
 	_qemu_cflags=""
 
 	# compile with high level of optimisation
-	_qemu_cflags+=" -O3"
+	if [ "${qemu_version_major}" -le 4 ] || [[ "${qemu_version_major}" -eq 5 && "${qemu_version_minor}" -lt 2 ]]; then
+		_qemu_cflags+=" -O3"
```

Comment:
why not just ignore the meson warning? what the default optimization?

Reply:
If we stick with `-O3` then this fix is require -> https://lists.gnu.org/archive/html/qemu-devel/2021-01/msg07262.html

By default QEMU is build with `-O2`. I'd like to understand if `-O3` is really required by Kata Containers.

Reply:
ok, this is is nice to have